packages/vaadin-element-mixin/vaadin-element-mixin.d.ts

```diff
@@ -1,5 +1,5 @@
-/// <reference path="custom_typings/vaadin-usage-statistics.d.ts" />
-/// <reference path="custom_typings/vaadin.d.ts" />
+import 'custom_typings/vaadin-usage-statistics.js';
+import 'custom_typings/vaadin.js';
```

Comment:
I'm not quite sure about this change, but TS doesn't encourage using `reference` tags and recommends using `imports` instead.

> Do not use a triple slash reference for custom_typings/vaadin-usage-statistics.d.ts, use `import` style instead.